### ISSUE TITLE
Error in innerRef types in 1.7.0

### ISSUE THREAD

**Issue by pgsandstrom** [Norserium/react-indiana-drag-scroll#45]
I cant use the typing in the release notes. The release notes say to use this:

```
const ref = useRef<HTMLElement>();
```

However, then ref gets the typing `MutableRefObject<HTMLElement | undefined>`. The expected typing for innerRef is `React.Ref<HTMLElement>` and this does not match.

These are the typing versions used for react:
```
    "@types/react": "16.9.53",
    "@types/react-dom": "16.9.8",
```

**Comment by pgsandstrom** [comment]
I should add that it can be fixed by using this instead:

```
  const ref = useRef<HTMLElement | null>(null)
```

However, that makes the proptypes mismatch, which results in an annoying error message in the console.

**Comment by Norserium** [comment]
@pgsandstrom could you provide the annoying error message? I doesn't see any of them.

**Comment by pgsandstrom** [comment]
Sure:

```
Warning: Failed prop type: invalid 'instanceof' operand expectedClass
```

Im using create react app and its shown in the browser console, to be clear.

**Comment by Norserium** [comment]
@pgsandstrom are you sure, that this error occurs inside this library? Could you send the screenshot of the console?

**Comment by pgsandstrom** [comment]
Here is a screenshot from my project. I'll try to find time in the coming days to reproduce the error in the simplest possible environment.
![screenshot](https://user-images.githubusercontent.com/1449130/96363416-8f97e480-1134-11eb-856d-6c10737338f2.png)

**Comment by pgsandstrom** [comment]
Okay, I managed to replicate it. It doesnt happen on the first render, so just click the button to get it:

https://codesandbox.io/s/react-indiana-drag-scroll-typescript-initial-scroll-innerref-forked-8n2yv

**Comment by pgsandstrom** [comment]
Here is a sandbox with the error. It doesnt happen on the first render, so just click the button:

https://codesandbox.io/s/react-indiana-drag-scroll-typescript-initial-scroll-innerref-forked-8n2yv?file=/src/index.tsx

**Comment by Norserium** [comment]
Thank you. I'm investigating the problem.

**Comment by Norserium** [comment]
@pgsandstrom this error is fixed in `1.7.2`.

**Comment by pgsandstrom** [comment]
The error message is gone. Good work and thank you very much!
